templates/default-file-path.inc

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
I'm not seeing where this is used in the build process, or how this fits in generally.

Reply:
This is a template file for manual install, it is possible there is a better location for it. This file appears to be built by the installer, but when deploying the website without the installer, as in the case of a manually deployed shared server this file is missing, but required to provide a working path to the users home directory for sqlite files when a user can not write to the main /var location.

Reply:
Would your purpose be better served by adding a new target, not included in the default 'dist' target, that performs a manual install from source?  You could specify the default file path on the ant command line as a property (-D or -propertyfile) along with the destination directory.  The build script could then instantiate the template in place, in the manner of banner.inc and generated-version.inc.

That would also address the dist.oneclick issue, as you would only need to build the linux-in-situ target, rather than the default target that tries to build all platforms' artifacts.